src/lib/dfp/merchandising-high-test.ts

```diff
@@ -0,0 +1,6 @@
+import { isInVariantSynchronous } from 'lib/experiments/ab';
+import { billboardsInMerchHigh } from 'lib/experiments/tests/billboards-in-merch-high';
+
+export const includeBillboardsInMerchHigh = () => {
```

Comment:
**Why wrap the variant check in this helper?**

We'll probably want to change this check to show billboards to all non-participations whilst the test is running, something like:

```typescript
!isInTest(billboardsInMerchHigh) || isInVariantSynchronous(billboardsInMerchHigh, 'variant');
```

and this will save us from having to do it in a number of different places.